samples/meta-body-tracking-sample/export_presets.cfg

```
@@ -248,3 +248,255 @@ khronos_xr_features/htc/hand_tracking=0
 khronos_xr_features/htc/eye_tracking=0
 pico_xr_features/eye_tracking=0
 magicleap_xr_features/hand_tracking=0
+
+[preset.1]
+
+name="Android XR"
+platform="Android"
+runnable=true
```

We've standardized on having the Meta Quest preset as the sole runnable Android preset in the repo, because currently it's currently the most popular AOSP-compatible headset

```suggestion
runnable=false
```

---

Done